src/index.js

```diff
@@ -0,0 +1,95 @@
+const apiKey = "8a15946d573d485bb8971932231610";
+const searchBtn = document.getElementById("searchBtn");
+
+searchBtn.addEventListener("click", function () {
+    const cityInput = document.getElementById("cityInput");
+    const cityName = cityInput.value;
+
+    if (cityName) {
+        getTodayWeatherData(apiKey, cityName);
+        getForecastWeatherData(apiKey, cityName);
+    } else {
+        alert("Please enter city name.");
+    }
+});
+
+function getTodayWeatherData(apiKey, city) {
+    const apiUrl = `https://api.weatherapi.com/v1/current.json?key=${apiKey}&q=${city}&aqi=no`;
+    fetch(apiUrl)
+        .then(response => response.json())
+        .then(data => {
```

Comment:
De ce nu ai folosit sintaxa cu `await`?

```suggestion
    const response = fetch(apiUrl);
    const data = response.json();
```